### PR TITLE
[FEATURE] Add show/hide snippets (browser)

### PR DESCRIPTION
**Pull request by Chalarangelo**
## Description

Snippets for showing and hiding elements, similar to jQuery's methods.

## What does your PR belong to?
- [ ] Website
- [x] Snippets
- [ ] General / Things regarding the repository (like CI Integration)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking improvement of a snippet)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have checked that the changes are working properly
- [x] I have checked that there isn't any PR doing the same
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: snippets/hide.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Hides all the elements specified.
+
+Use the spread operator (`...`) and `Array.forEach()` to apply `display: none` to each element specified.
+
+```js
+const hide = (...el) => [...el].forEach(e => e.style.display = 'none');
+```
+
+```js
+hide(document.querySelector('img')); // Hides all <img> elements on the page
```
Comment (skatcat31) [inline snippets/hide.md]: querySelectorAll should be used to hide all. querySelector returns first found

File: snippets/show.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+Shows all the elements specified.
+
+Use the spread operator (`...`) and `Array.forEach()` to clear the `display` property for each element specified.
+
+```js
+const show = (...el) => [...el].forEach(e => e.style.display = '');
+```
+
+```js
+show(document.querySelector('img')); // Shows all <img> elements on the page
```
Comment (skatcat31) [inline snippets/show.md]: see other comment

File: snippets/hide.md (hunk 1 above)
Comment (Chalarangelo) [reply snippets/hide.md]: Fixed both!

**Comment by skatcat31** [comment]
timing is an annoyance when trying to approve and then request changes XD

**Pull request merged**